pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -1242,19 +1246,20 @@ impl RemoteTimelineClient {
         Ok(())
     }
 
-    pub(crate) fn schedule_barrier(self: &Arc<Self>) -> anyhow::Result<()> {
+    pub(crate) fn schedule_barrier(self: &Arc<Self>, initial_barrier: bool) -> anyhow::Result<()> {
```

Comment:
it would be better to have `schedule_barrier_initial` instead of adding a boolean parameter everywhere.